src/modules/launcher/Wox/PublicAPIInstance.cs

```diff
@@ -138,9 +143,19 @@ public void PushResults(Query query, PluginMetadata plugin, List<Result> results
             });
         }
 
+        public Theme GetCurrentTheme()
+        {
+            return _themeManager.GetCurrentTheme();
+        }
+
         #endregion
 
-        #region Private Methods
+        #region Protected Methods
+
+        protected void OnThemeChanged(Theme oldTheme, Theme newTheme)
+        {
+            ThemeChanged?.Invoke(oldTheme, newTheme);
```

Comment:
[clarification] Is there any time the event handler would be null?

Reply:
The event handler would be null if no plugin register for ThemeChanged Event.